### ISSUE TITLE
fork() call generates error

### ISSUE THREAD

**Issue by vital1** [Mellanox/libvma#87]
Suppose, we have the following programm on C++:
```
#include <iostream>
#include <unistd.h>
#include <system_error>
#include <cerrno>

using namespace std;

int main()
{
	pid_t pid = fork();
	if (pid < 0)
		throw system_error(errno, std::system_category());
	else if (pid == 0)
	{
		cout << "child" << endl;
		sleep(1);
		_exit(0);
	}
	else
	{
		cout << "parent" << endl;
		sleep(2);
	}
	return 0;
}
```
Without libvma.so it works as it should be: it prints "parent" and "child" to the console.
With the latest version of libvma.so we have the following output:
```
 VMA INFO: ---------------------------------------------------------------------------
 VMA INFO: VMA_VERSION: 8.0.2-0 Development Snapshot built on May 16 2016 16:54:27
 VMA INFO: Cmd Line: ./a.out
 VMA INFO: Current Time: Mon May 16 18:40:13 2016
 VMA INFO: Pid: 24648
 VMA INFO: OFED Version: MLNX_OFED_LINUX-3.1-1.0.3:
 VMA INFO: Architecture: x86_64
 VMA INFO: Node: host9
 VMA INFO: ---------------------------------------------------------------------------
 VMA INFO: Log Level                      INFO                       [VMA_TRACELEVEL]
 VMA INFO: ---------------------------------------------------------------------------
parent
./a.out: symbol lookup error: /usr/lib64/libvma.so: undefined symbol: _Z14get_env_paramsv
```
There was no such error in the previous version of libvma; we have the following output for previous version of libvma:
```
 VMA INFO: ---------------------------------------------------------------------------
 VMA INFO: VMA_VERSION: 8.0.1-0 Development Snapshot built on May 16 2016 18:46:37
 VMA INFO: Cmd Line: ./a.out
 VMA INFO: Current Time: Mon May 16 18:49:49 2016
 VMA INFO: Pid: 32748
 VMA INFO: OFED Version: MLNX_OFED_LINUX-3.1-1.0.3:
 VMA INFO: Architecture: x86_64
 VMA INFO: Node: host9
 VMA INFO: ---------------------------------------------------------------------------
 VMA INFO: Log Level                      INFO                       [VMA_TRACELEVEL]
 VMA INFO: ---------------------------------------------------------------------------
parent
child
```

**Comment by rosenbaumalex** [comment]
@OphirMunk is this issue fixed with your #88 PR/commit?

**Comment by OphirMunk** [comment]
@vital1 and @rosenbaumalex - yes. Issue is fixed in  #88 PR/commit. Please pull latest master

**Comment by rosenbaumalex** [comment]
seems resolved.
no new activity for some time... closing
